org.eclipse.jdt.core.compiler.batch/src/org/eclipse/jdt/internal/compiler/util/JrtFileSystem.java

```diff
@@ -0,0 +1,322 @@
+package org.eclipse.jdt.internal.compiler.util;
```

Comment:
Please add a Copyright header for this file. It should consider when and by whom `JrtFileSystem` was added.